scripts/globals/mobskills/dark_spore.lua

```diff
@@ -20,7 +20,11 @@ mobskillObject.onMobWeaponSkill = function(target, mob, skill)
     local typeEffect = xi.effect.BLINDNESS
     xi.mobskills.mobStatusEffectMove(mob, target, typeEffect, 15, 3, 120)
 
-    local dmgmod = xi.mobskills.mobBreathMove(mob, target, 0.25, 2, xi.magic.ele.DARK, 800)
+    local cap = 800
+    if mob:isInDynamis() then
```

Comment:
Need a check for owned by a player, though I doubt anyone would actually bring a Funguar Familiar to dynamis..